metrics/README.md

```diff
@@ -0,0 +1,63 @@
+# Clear Containers metrics tests
+
+This section contains a collection of scripts designed to measure the performance
+and responsiveness of the different components which form Clear Containers.
+
+## Metrics tests - content
+
+The tests are organized in the following sections:
+
+| Section   | Short description
+| ----------| --------------------------------------------------------------------------------
+| density   | Measure the amount of memory used by N containers under different configurations.
+| network   | Measure a set of network essentials such as: bandwidth, jitter, latency, etc.
+| storage   | Measure storage bandwith using different configurations.
+| time      | Measure the latency that a container[s] completing/executing a defined workload.
```

Comment:
Sorry @MarioCarrilloA I should have been more clear. Please choose the best verb. Completing OR executing

Reply:
oh sorry my bad, thanks